src/scheduler/pbs_sched.c

```diff
@@ -938,6 +937,7 @@ main(int argc, char *argv[])
 	int num_cores;
 	char *endp = NULL;
 	pthread_mutexattr_t attr;
+	static int update_svr = 1;
```

Comment:
This doesn't need to be static.  When main ends, so does the scheduler.

Reply:
Done